en/known-issues.md

```diff
@@ -1,37 +1,90 @@
 ---
 title: Known Issues - Design System
 _description: The issues and limitations that we are aware of with the guidance how to avoid or recover in case you run into them.
-_keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angular, Sketch to Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch HTML, Sketch to HTML, Sketch UI kits
+_keywords: Design Systems, Design Systems UX, UI kit, Figma, Sketch, Ignite UI for Angular, Figma to Angular, Sketch to Angular, Angular, Angular Design System, Export code from Figma, Export code from Sketch, Design Kits for Angular, Figma HTML, Figma to HTML, Sketch HTML, Sketch to HTML, Sketch UI kits, Figma UI kits
 ---
 
 # Known Issues
 
-This topic describes the issues and limitations we have run into or observed when using the Indigo Design Sketch Libraries. Where applicable we will be providing the necessary guidance to avoid the occurence of issues or recover from them once they occur.
+This topic describes the issues and limitations we have run into or observed when using the Indigo Design Libraries in Figma, Sketch and AdobeXD. Where applicable we will be providing the necessary guidance to avoid the occurrence of issues or recover from them once they occur.
 
-## Detached Symbols
+### General
+
+#### Detached Symbols
 
 As you may have already seen in the other topics detaching or ungrouping symbols usually results in the loss of capability for code generation. However, a few components like the List and the Grid are basically repeaters of items and cells and come in a more generic form that is supposed to be detached to allow you to create the number of items, rows and columns that you would need. As we enhance individual items with functionality in the upcoming versions of the libraries, such updates should propagate easily, however enhancements made to the collection of items i.e. `List/One-line Item List` or `Grid/Comfortable`, for example, will not propagate to the detached instances of these components.
 
-While `Detach from Symbol` is something you may eventually resort to in Sketch, with Adobe XD remember that you can even add/remove layers in instances of a symbol. This is especially useful when content is templatable e.g. in a list item you may want to add a Chip, which is not there in any of the template options.
+While `Detach from Symbol` is something you may eventually resort to in Sketch, with Adobe XD remember that you can even add/remove layers in instances of a symbol. This is especially useful when content is templatable e.g. in a list item you may want to add a Chip, which is not there in any of the template options. And in Figma detaching is less needed and its possible to modify the inner content.
```

Comment:
And in Figma detaching is less needed and its possible to modify the inner content.
-> Did you mean in XD? Only Xd allows completely removing or moving elements within a component Instance without needing to detach it. In Figma you can only hide some of the content and nothing else.